### DISCUSSION TITLE
Is there any Kepler receiver?

### DISCUSSION THREAD

**Discussion by alan-cueva** [open-telemetry/opentelemetry-collector-contrib#29533] [Q&A]
Hello everyone,
I am new in OTEL and newer in Kepler (Kubernetes Efficient Power Level Exporter) and I need to create a collector that uses Kepler as a receiver, but there is no implementation of this. What can I do? Is there any tutorial of how to develop my own receiver?
Thanks in advance!

**Comment by frzifus** [comment]
Hi @alan-cueva, Kepler is instrumented using the prometheus sdk and offers a prometheus endpoint. That can be scraped using the opentelemetry-collector [Prometheus Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/632894c95d20d0a94c55e2bf18f2429bd861a350/receiver/prometheusreceiver/README.md).

AFAIK there was also some work to report otlp using the prometheus sdk. But I am not sure about the state.

Otherwise follow this issue:  https://github.com/sustainable-computing-io/kepler/issues/659
